package.json

```
@@ -127,7 +129,7 @@
     "@typescript-eslint/eslint-plugin": "^4.22.1",
     "@typescript-eslint/parser": "^4.22.1",
     "@wordpress/browserslist-config": "^4.0.0",
```

```suggestion
    "@wordpress/browserslist-config": "^4.1.0",
```
Should we have the same version as in the premium plugin?